src/controllers/UserController.ts

```diff
@@ -48,9 +48,24 @@ export class UserController {
 		}
 	}
 
+	public async resendEmail(req: Request, res: Response, next: NextFunction): Promise<void> {
+		try {
+			const user = await this.userService.registerUser(req.body);
```

Comment:
This is calling the wrong userService method

Reply:
Typed the wrong name, changed it in latest commit